wagtail/admin/templates/wagtailadmin/reports/workflow_tasks.html

```diff
@@ -1,74 +1,60 @@
-{% extends "wagtailadmin/base.html" %}
-{% load i18n l10n wagtailadmin_tags wagtailui_tags %}
+{% extends 'wagtailadmin/reports/base_report.html' %}
+{% load i18n wagtailadmin_tags %}
 
-{% block titletag %}{% trans "Workflows" %}{% endblock %}
-
-{% block content %}
-    {% trans "Workflows" as title_str %}
-
-    <header role="banner">
-        <div class="row nice-padding">
-            <div class="left">
-                <div class="col header-title">
-                    <h1 class="icon icon-{{ header_icon }}">Workflows</h1>
-                </div>
-            </div>
-            <div class="right">
-                <div class="addbutton">
-                    <a href="{% url 'wagtailadmin_reports:workflow' %}" class="button">By Workflow</a>
-                </div>
-            </div>
-        </div>
-    </header>
+{% block actions %}
+    <div class="addbutton">
+        <a href="{% url 'wagtailadmin_reports:workflow' %}" class="button">By Workflow</a>
+    </div>
+    {{ block.super }}
+{% endblock %}
 
-    <div class="nice-padding">
-        {% if object_list %}
-            <table class="listing">
-                <thead>
+{% block results %}
```

Comment:
This report doesn't seem to be paginated (unless I'm missing something) - I think that would be worth adding if not.

Reply:
(and I think same with the other)

Reply:
Pagination is handled by Django's `BaseListView`, which `ReportView` inherits from:

https://github.com/django/django/blob/master/django/views/generic/list.py#L118

Reply:
Sorry, I should have been clearer - I meant frontend pagination tags like in `BasePageReport`, eg https://github.com/wagtail/wagtail/blob/860588c80763059c75abe7512dd6be597abed4b3/wagtail/admin/templates/wagtailadmin/reports/base_page_report.html

Reply:
Ah, I see, yep - that `{% paginate %}` tag probably ought to be moved out into base_report...

Reply:
Fixed in 996d4806f1a9f07d7691710414c30952be01016a